tests/unit_tests/enrichment_test.py

```diff
@@ -0,0 +1,112 @@
+"""Test that the enrichment functions are working correctly"""
+from gopher import enrichment, mannwhitneyu
+import pandas as pd
+import random
+from scipy import stats
+import numpy as np
+
+
+def test_entire_enrichment_analysis():
+    """Check that the test enrichment function returns a dataframe.
+    This test will run the full GO enrichment on a dataset and takes
+    roughly 3.5 minutes to run."""
+    df = pd.read_csv("../data/df.csv")
+    df.set_index("Protein", inplace=True)
+    result = enrichment.test_enrichment(df)
+    assert type(result) is pd.DataFrame
```

Comment:
**todo:** Let's make this have a shorter runtime.
In general, I like for unit tests to be very quick (no more than a second each). This helps ensure that folks actually run them before contributing.

**todo:** Instead of adding a data file to the repo to use, we should simulate an input file using a [pytest fixture](https://docs.pytest.org/en/6.2.x/fixture.html). 
Please ask me about it if you have questions!

Reply:
I did add this as a fixture, but I would like you to look over it again because I am not sure if my solution is a technically correct one.

Reply:
No problem - I'll take a look!